assets/app/scripts/controllers/builds.js

```diff
@@ -50,6 +52,12 @@ angular.module('openshiftConsole')
         }
       }
 
+      // Scroll to anchor on first load if location has a hash.
+      if (!action && $location.hash()) {
+        // Wait until the digest loop completes.
+        setTimeout($anchorScroll, 10);
```

Comment:
We're not changing any data, so no need to trigger a digest cycle. We had some performance issues with `$timeout` and `$interval` and hawtio-nav, so I try to avoid them now if not necessary.